jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -629,6 +629,12 @@
         jenkins-timeout: 170
         timeout: 70
         trigger-job: 'ci-kubernetes-build'
+    - kubernetes-e2e-gce-gci-qa-m60:
```

Comment:
@wonderfly ok sgtm, I already have a script does the migration job, will send a migration PR for all gci-qa job later.